lib/modules/neverEndingReddit.js

```diff
@@ -317,8 +317,7 @@ function createLoaderWidget(startPromise, donePromise) {
 	});
 
 	const removeWidget = () => widget.remove();
-	if (process.env.BUILD_TARGET !== 'edge') donePromise.finally(removeWidget);
```

Comment:
I think this line should be copied, not the bottom line.

Reply:
Ah missed the !. Will fix now.